Sources/FoundationEssentials/String/String+Internals.swift

```diff
@@ -170,8 +171,9 @@ extension String {
         let bLeadingSlash = false
 #endif
 
-        var mut: Substring =
+        var mut: String =
             Substring(self.utf8[self.utf8.index(self.utf8.startIndex, offsetBy: bLeadingSlash ? 1 : 0)...])
+                .replacing(._slash, with: ._backslash)
```

Comment:
We probably need to guard this with `os(Windows)`, too

Reply:
Yeah, that is a good point.